smolyay/_growth.py

```diff
@@ -0,0 +1,144 @@
+import numpy
+
+
+def clenshaw_curtis_exponential_growth(num_levels):
```

Comment:
Since these are functions, they should have action names. They might need be shortened a bit though. Maybe like:

* `make_clenshaw_curtis_level_sizes`
* `make_slow_clenshaw_curtis_level_sizes`
* `make_trigonometric_level_sizes`

?

Reply:
I want to keep exponential as part of the name to match Burkardt's nomenclature.

Reply:
I can't recall. Burkardt is the main source of information about the subject. I'll rename them according to the classes.